### PR TITLE
Tests and publish package with GitHub Actions

### PR DESCRIPTION
**Pull request by jkfran**
### Done
- Remove CircleCI config
- Convert README file to Markdown
- GitHub Action: Tests and build check for new PRs
- GitHub Action: Build package and publish to PyPI when the branch `master` is updated

### QA
Make sure the checks with the "Tests /" prefix pass

### Issue
https://github.com/canonical-web-and-design/web-squad/issues/2094

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-and-publish:
```
Comment (nottrobin) [inline .github/workflows/publish.yml]: Is this consistent with the naming we've used elsewhere? Couldn't it just be `publish`?

File: setup.py [hunk 2]
```diff
@@ -4,21 +4,16 @@
...
-    ),
-    long_description=open('README.rst').read(),
-    install_requires=[
-        "mistune",
-        "python-frontmatter",
-        "django",
-    ],
+    description=("Shared Django views for use in Webteam apps"),
+    long_description=open("README.md").read(),
+    long_description_content_type="text/markdown",
```
Comment (nottrobin) [inline setup.py:15]: :champagne: :fireworks:

File: setup.py [hunk 3]
```diff
@@ -4,21 +4,16 @@
...
-    description=(
-        "Shared Django views for use in Webteam apps"
-    ),
-    long_description=open('README.rst').read(),
-    install_requires=[
-        "mistune",
-        "python-frontmatter",
-        "django",
-    ],
+    description=("Shared Django views for use in Webteam apps"),
```
Comment (nottrobin) [inline setup.py]: Doesn't need brackets

**Review by nottrobin — APPROVED** [review]
Looks good. Just some small comments, but whether you address them or not, :+1:

File: .github/workflows/publish.yml (hunk 1 above)
Comment (jkfran) [reply .github/workflows/publish.yml]: Yes, and it will be :smile:

File: setup.py (hunk 2 above)
Comment (jkfran) [reply setup.py:15]: :champagne:

File: setup.py (hunk 3 above)
Comment (jkfran) [reply setup.py]: You right :+1:

**Pull request merged**